utils/firestore.js

```diff
@@ -427,29 +428,56 @@ const removeParticipantsDataDestruction = async () => {
             const participant = doc.data();
             const millisecondsInDay = 24 * 60 * 60 * 1000;
```

Comment:
This can be moved out of loop. There's no need to calculate it in every iteration.

Reply:
Solved